pipeline.yml

```diff
@@ -592,8 +487,7 @@ jobs:
     public: true
     plan:
       - get: instana-agent-docker-git
-        trigger: true
-        passed: [ prepare-build-bundle ]
+        passed: [ self-update, prepare-build-bundle ]
```

Comment:
Not sure why we would need the `self-update` everywhere as being 'passed'?
After the first step, we know it already passed the `self-update` so its not needed for every next step I believe.
Also with `self-update` we only update the pipeline itself. If it checks for only updates to `/ci/` then its even meaningless to add because changes are unrelated.